src/app/wallets/update-pending-payments.ts

```diff
@@ -89,14 +90,24 @@ const updatePendingPayment = async ({
         return
       }
 
-      const settled = await ledgerService.settlePendingLnPayments(paymentHash)
+      const settled = await ledgerService.settlePendingLnPayments({
+        paymentHash,
+      })
       if (settled instanceof Error) {
         paymentLogger.error(
           { error: settled },
           "we didn't have any transaction to update",
         )
         return settled
       }
+      const persistedPayment = await LnPaymentsRepository().update(lnPaymentLookup)
+      if (persistedPayment instanceof Error) {
+        paymentLogger.error(
+          { error: lnPaymentLookup },
+          "we couldn't update payment data to our database",
+        )
+        return persistedPayment
```

Comment:
we should only log the update error but we cant compromise reimburseFee or revert transaction. if you want to keep the return error please move after reimburseFee and revertTransaction

Reply:
Removed the error and opted to log only. We can probably double-check that this is added correctly in whichever job we will be deleting payments from lnd's database from